source/features/clean-issue-filters.tsx

```diff
@@ -40,18 +30,26 @@ function removeParent(element?: Element): void {
 	}
 }
 
-async function init(): Promise<void> {
-	const {projects, milestones} = await getCounts();
+async function hideMilestones(): Promise<void> {
+	const hasMilestones = parseInt(select('[data-selected-links^="repo_milestones"] .Counter')?.textContent, 10) > 0;
 
-	if (projects === 0) {
-		elementReady('[data-hotkey="p"]').then(removeParent);
+	if (hasMilestones) {
```

Comment:
Maybe this can be its own function:

```js
function hasLocalCounter(selector: string): boolean {
	return Number(select(`${selector} .Counter`)?.textContent!.trim()) > 0;
}
```
And then
```suggestion
	if (hasLocalCounter('[data-selected-links^="repo_milestones"]')) {
```

Reply:
Applied